Sources/Sentry/Public/SentryMechanism.h

```diff
@@ -35,6 +35,12 @@ SENTRY_NO_INIT
  */
 @property (nonatomic, copy) NSNumber *_Nullable handled;
 
+/**
+ * An optional flag indicating a synthetic exception. For more info visit
+ * https://develop.sentry.dev/sdk/event-payloads/exception/#exception-mechanism.
+ */
+@property (nonatomic, copy, nullable) NSNumber *synthetic;
```

Comment:
I think this should be simply a `BOOL`, as that's what this flag is in the end.

Reply:
I understand that we're using NSNumber to have a nullable boolean, but is this really necessary? 
Is it "undefined" really an option?

Reply:
Yes, it's undefined. We don't send it currently. I would like to keep it, so the payload is smaller.